docs/frontmatter.md

```diff
@@ -54,6 +54,25 @@ project:
   open_access: true
 ```
 
+(composing-myst-yml)=
+:::{note} Composing multiple `.yml` files
```

Comment:
If the TOC PR gets in before this one is finished:

- https://github.com/executablebooks/mystmd/pull/1188

I'd recommend using that as an example here. I think a common use-case will be to have a separate `toc.yml` file and use this feature to bring it in (for communities that have a _lot_ of TOC content and thus don't want to have to scroll through a huge list of pages in their config file)

Reply:
That makes sense - I'll update, depending on the order these come in...

I also suspect we will be scattering this all around the docs. E.g. where we talk about math macros, we can say "reference external macros by..." and where we talk about the toc, "reference external toc by..." etc. So hopefully this one little section won't be the only mention in the docs, long term.